front/src/styles/scss/applications/stdcm/_home.scss

```
@@ -194,13 +201,17 @@
           display: flex;
           flex-direction: column;
           padding: 16px 29px 32px 29px;
-          margin-top: 12px;
+          margin-bottom: 32px;
 
           span {
             color: #eaa72b;
             align-self: center;
             margin-bottom: 6px;
           }
+          & ul {
+            list-style: disc;
+            margin-left: 15px;
```

We need to reset the `margin-bottom` as well, otherwise the cumulated `margin-bottom` will be higher than the 32px of the mockup.
```suggestion
            margin-left: 15px;
            margin-bottom: 0;
```

---

done